src/__tests__/lossy.js

```diff
@@ -0,0 +1,83 @@
+import ava from 'ava';
+import parser from '../index';
+
+export const parse = (input, options, transform) => {
+    return parser(transform).process(input, options).result;
+};
+
```

Comment:
AVA now has support for test macros, which is pretty much exactly what you're doing here (but "unofficially"). So, instead of:

```js
export const testLossy = (spec, input, expected) => {
    let result = parse(input, {lossless:false});

    ava(`${spec} (toString)`, t => {
        t.deepEqual(result, expected);
    });
};

testLossy('combinator, descendant - single', '.one .two', '.one .two');
```

We can do this instead:

```js
export const testLossy = (t, input, expected) => {
    let result = parse(input, {lossless:false});
    t.deepEqual(result, expected);
};

ava('combinator, descendant - single', testLossy, '.one .two', '.one .two');
```

Which is a lot cleaner. What do you think? 😄